pacman-hook-kernel-install/60-kernel-install-remove.hook

```diff
@@ -14,6 +14,6 @@ Target = usr/lib/modules/*/vmlinuz
 
 [Action]
 Description = Removing kernel and initrd from $BOOT... (kernel-install)
-When = PreTransaction
+When = PostTransaction
```

Comment:
Rationale?

Reply:
See second bullet in the pull request description 🙂

Reply:
> See second bullet in the pull request description

I missed that, sorry.

The previous maintainer used the same priority 90 in the pacman hooks. I assume PreTransaction was used to make sure remove runs before add. Now it is not important since I moved it to priority 60.

Reply:
> I missed that, sorry.

No worries, I figured 🙂